airbyte-integrations/connectors/destination-redshift/src/test-integration/java/io/airbyte/integrations/destination/redshift/RedshiftS3StagingInsertDestinationAcceptanceTest.java

```diff
@@ -4,15 +4,51 @@
 
 package io.airbyte.integrations.destination.redshift;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
 import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.airbyte.cdk.integrations.destination.record_buffer.FileBuffer;
 import io.airbyte.commons.io.IOs;
 import io.airbyte.commons.json.Jsons;
 import java.nio.file.Path;
+import org.junit.jupiter.api.Test;
 
-public class RedshiftS3StagingInsertDestinationAcceptanceTest extends RedshiftStagingS3DestinationAcceptanceTest {
+/**
+ * Integration test testing {@link RedshiftStagingS3Destination}. The default Redshift integration
+ * test credentials contain S3 credentials - this automatically causes COPY to be selected.
+ */
+public class RedshiftS3StagingInsertDestinationAcceptanceTest extends RedshiftDestinationAcceptanceTest {
 
   public JsonNode getStaticConfig() {
     return Jsons.deserialize(IOs.readFile(Path.of("secrets/config_staging.json")));
   }
 
+  /*
+   * FileBuffer Default Tests
+   */
+  @Test
+  public void testGetFileBufferDefault() {
```

Comment:
these should be a unit test :sob: there's no reason to run through all the BeforeEach/AfterEach stuff for this. (DAT setup/teardown takes nontrivial time, running these in DATs is just wasteful)

Reply:
I also pulled out the connection checks as unit tests 👀